### PR TITLE
New version: Cwtch.Cwtch version 1.14.7

### PR DESCRIPTION
**Pull request by OfficialEsco**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/145506)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
```
Enter a pull request number: #145506

Manifest validation succeeded.
Found Cwtch [Cwtch.Cwtch] Version 1.14.7
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://git.openprivacy.ca/cwtch.im/cwtch-ui/releases/download/v1.14.7/cwtch-installer-v1.14.7.exe
  ██████████████████████████████  52.5 MB / 52.5 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : Cwtch
Publisher     :
Version       :
ProductCode   : Cwtch
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Cwtch
Uninstall     : winget uninstall --id "Cwtch"

Press any key to continue . . .
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 870.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 871.)

File: manifests/c/Cwtch/Cwtch/1.14.7/Cwtch.Cwtch.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Cwtch.Cwtch
+PackageVersion: 1.14.7
+PackageLocale: en-US
+Publisher: Cwtch
+PrivacyUrl: https://openprivacy.ca/
+PackageName: Cwtch
```
Comment (stephengillie) [inline manifests/c/Cwtch/Cwtch/1.14.7/Cwtch.Cwtch.locale.en-US.yaml:9]:
```suggestion
PackageName: Cwtch
PackageUrl: https://git.openprivacy.ca/cwtch.im/cwtch-ui
```

**Comment by stephengillie** [comment]
Should [this `PackageUrl`](https://github.com/microsoft/winget-pkgs/pull/145506#discussion_r1535631134) be used? Other than that, this PR looks ready to merge.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**